introduction.md

```diff
@@ -2,3 +2,9 @@
 > I am learning forking and cloning
 > Awesome!!
 **My favorite Code school path is definitely Electives**
+
+**My deadly skills:**
+
+* CSS
+* PHP
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?